app/timetables/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
-# Register your models here.
+from app.timetables.models import Timetable
+
```

Comment:
@andela-ooshodi adding `from future import absolute_import` to the top of the file may not be necessary as we are using python3.